Pydantic 2.0 deprecations

Pydantic 2 was released a few days ago... some of our tests are now failing with 
```
.env/lib/python3.11/site-packages/pydantic/_internal/_config.py:206: in prepare_config
    warnings.warn(DEPRECATION_MESSAGE, DeprecationWarning)
E   pydantic.warnings.PydanticDeprecatedSince20: Support for class-based `config` is deprecated, use ConfigDict instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.0.1/migration/
        config     = <class 'jsf.schema_types.enum.JSFEnum.Config'>
```
(pytest is set up to treat warnings as errors)

Until deprecated usages are addressed, would it be possible to specify the requirement on `pydantic` as `>=1.10.4,<2` (instead of just `>=1.10.4`)?

--- a/jsf/requirements.txt
+++ b/jsf/requirements.txt
@@ -1,6 +1,6 @@
 faker>=15.3.4
 jsonschema>=4.17.3
-pydantic>=1.10.4
+pydantic>=1.10.4,<2.0.0
 rstr>=3.2.0
 smart-open[http]>=6.3.0
 

